src/components/Chart/plugins/chart.triggerLevel.js

```diff
@@ -89,19 +91,15 @@ const plugin = {
         const { label } = this.getCoords(chartInstance) || {};
         if (!label) return;
         chartInstance.triggerLine.y = evt.layerY;
+        const {
+            scales: { yScale },
+            options: { sendTriggerLevel },
+        } = chartInstance;
+        sendTriggerLevel(yScale.getValueForPixel(chartInstance.triggerLine.y));
```

Comment:
Sending the trigger level is a huge change for PPK1, make sure the hardware does not get updated by pointer movement here. This was the reason why there was no continuous update in the SidePanel before.

For PPK1 the SidePanel value will have to be out of sync with the firmware until you release the pointer, but that is the only way it is not spammed.

Reply:
Great catch, I wasn't aware of this. I will add some logic to handle PPK1 seperately. Then we can discuss if we want to handle this differently once people can try it out and get a feel for it themselves.

Reply:
I think we can have both: Keep the SidePanel value in sync and do not overwhelm the PPK1 by splitting up `updateTriggerLevel` into the part that updates the state (which can be done on every mouse move) and the part that sends the new level to the device (which we then do only on pointer leave or up). In the input field both can be done on complete together.

Reply:
Good idea!